mdn/kumascript
# Redesigned compat table macro

To test this, you will need:
- Kuma with https://github.com/mozilla/kuma/pull/4436.
- A waffle flag called `compat_data` enabled.
- A test page with some compat data calls. (TBD, I'll post one here).

How this works:
- Compat.ejs is mostly untouched and additionally loads CompatBeta.ejs, but all HTML that is generated by CompatBeta is hidden.
- Only if the waffle flag is on, the CompatBeta output is shown, and the Compat output is hidden.
- This allows us to do A/B or user testing.
- Once we are done with testing, CompatBeta.ejs will be copied into Compat.ejs, and CompatBeta.ejs can be deleted. (On the pages, no macro calls need to be updated. Before, during and after the beta the {{compat("something.something")}} calls are the same).

Testing:
- run tests with `make test-macros` in the kumascript dir, it might require a `make build-kumascript` in the kuma dir before.
- The CompatBeta output is tested by test-compat.js and compat data fixtures.
- I've added basic tests, we should expand this going forward. There are many cases in compat data.
